src/rest/main/main.go

```diff
@@ -56,7 +56,7 @@ func init() {
 	flag.StringVar(&caFile, "cacert", "", "CA certificate for client certificate validation")
 	flag.StringVar(&cliCAFile, "clicacert", "", "CA certificate for CLI client validation")
 	flag.Var(clientAuth, "client_auth", "Client auth mode(s) - <none,cert,jwt,password|user(deprecated)> default: password,jwt")
-	flag.BoolVar(&server.RbacDisable, "rbac-disable", false, "Disable RBAC")
+	flag.BoolVar(&server.RbacDisable, "rbac_disable", false, "Disable RBAC (Default Enabled)")
```

Comment:
Haha, so the language here is confusing. When you say "default enabled", do you mean that RBAC is enabled by default, or that the flag (to disable RBAC) is enabled by default?

Reply:
Ya it is a bit confusing, but yes that's what I meant. Note that you don't pass true|false to this flag, you either pass the flag --rbac_disable or you don't pass it.